main.js

```diff
@@ -1,3 +1,6 @@
+import { data } from "autoprefixer";
```

Comment:
Is "data" used anywhere? If not, you can remove the import.

Reply:
Cred ca era folosit in proiectul initial, dar eu nu-l mai folosesc. Am sters toate importurile pe care nu le folosesc.